src/requirements.in

```
@@ -8,7 +8,8 @@ django-postgres-unlimited-varchar == 1.1.0
 django-gisserver == 0.5
 djangorestframework == 3.11.0
 djangorestframework-gis == 0.15
-amsterdam-schema == 0.1.2
+amsterdam-schema-tools == 0.4.1
+amsterdam-schema-tools[django] == 0.4.1
```

```suggestion
amsterdam-schema-tools[django] == 0.4.1
```

alleen de laatste is dan nodig.